static/app/views/onboarding/index.tsx

```diff
@@ -191,6 +191,14 @@ function Onboarding(props: Props) {
     );
   };
 
+  const jumpToSetupProject = useCallback(() => {
+    const nextStep = onboardingSteps.find(({id}) => id === 'setup-docs');
+    if (!nextStep) {
```

Comment:
Do we expect this to ever happen? If we dont, I would opt to throw here so we dont silently break the user flow (e.g. user clicking on a button that does nothing)

Reply:
@JonasBa It shouldn't happen...but silently breaking here is preferable. It just means the user has to do project selection. It's better than blowing up to the user.

Reply:
It's not from a button click fwiw

Reply:
Ah yes, my bad.